spanner/api/Spanner.Samples.Tests/UpdateUsingDmlReturningAsyncPostgreTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Spanner.Data;
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class UpdateUsingDmlReturningAsyncPostgreTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public UpdateUsingDmlReturningAsyncPostgreTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public async Task TestUpdateUsingDmlReturningAsyncPostgre()
+    {
+        await InsertDataAsync();
+        var sample = new UpdateUsingDmlReturningAsyncPostgreSample();
+        var updatedMarketingBudgets = await sample.UpdateUsingDmlReturningAsyncPostgre(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.PostgreSqlDatabaseId);
+
+        Assert.Single(updatedMarketingBudgets);
+        Assert.Equal(20000, updatedMarketingBudgets[0]);
+    }
+    private async Task InsertDataAsync()
+    {
+        var batchCommand = _spannerFixture.PgSpannerConnection.CreateBatchDmlCommand();
+        batchCommand.Add("INSERT INTO Singers (SingerId, FirstName, LastName) VALUES (14, 'Kishore', 'Kumar')");
```

Comment:
Probably not something to change now, but having all this data scattered around makes me nervous. I'd prefer to have all the names and IDs in the fixture. Something to chat with @amanda-tarafa about when she'd back.

Reply:
Yes, I agree. I'll keep a note of it.